test/support/services_and_information_helpers.rb

```diff
@@ -0,0 +1,21 @@
+module ServicesAndInformationHelpers
+  def stub_services_and_information_content_item
+    content_store_has_item("/government/organisations/hm-revenue-customs/services-information",
```

Comment:
Can you use the schemas gem for to generate this?

https://alphagov.github.io/govuk_schemas_gem/GovukSchemas/RandomExample.html#merge_and_validate-instance_method

It'll validate the data you put in the content store, so we can be certain that the stubbed data is real.